### PR TITLE
Prepare CHIPs for GA

### LINKED ISSUE #28070: Remove CHIPS Beta Guard Changes
Added in PR:
- https://github.com/OpenLiberty/open-liberty/pull/27809

Changed:
- BuildPaths: 
  - dev/com.ibm.ws.webcontainer.cors/bnd.bnd
  - dev/com.ibm.ws.webcontainer/bnd.bnd
  
Any files with ProductInfo 
-   src/com/ibm/wsspi/genericbnf/KeyMatcher.java
-   src/com/ibm/ws/http/channel/internal/HttpChannelConfig.java
-   src/com/ibm/ws/http/channel/internal/cookies/CookieUtils.java
-  and others ( security code)

### PR DESCRIPTION
**Pull request by volosied**
for #27405

fixes https://github.com/OpenLiberty/open-liberty/issues/28070

Depends on approvals

### CODE REVIEW

**Comment by volosied** [comment]
#build #fat.buckets.to.run=com.ibm.ws.webcontainer.servlet.4.0_fat

File: dev/com.ibm.ws.session/resources/OSGI-INF/l10n/metatype.properties [hunk 1]
```diff
@@ -59,7 +59,7 @@ cookie.same.site.none=Set the session cookie SameSite attribute value to None
 cookie.same.site.strict=Set the session cookie SameSite attribute value to Strict
 
 # The word Partitioned should not be translated
-cookie.partitioned=Include a Partitioned attribute in session cookies
+cookie.partitioned=Partiton the session cookie
```
Comment (volosied) [inline dev/com.ibm.ws.session/resources/OSGI-INF/l10n/metatype.properties]: Changed due to the error: 
```junit.framework.AssertionFailedError: The label for attribute cookiePartitioned of type com.ibm.ws.session contains the banned word attribute in the label "Include a Partitioned attribute in session cookies".```

**Comment by volosied** [comment]
#build

**Comment by volosied** [comment]
#build #spawn.fullfat.buckets=io.openliberty.webcontainer.servlet.6.0.internal_fat,com.ibm.ws.webcontainer.servlet.4.0_fat

File: dev/com.ibm.ws.session/resources/OSGI-INF/l10n/metatype.properties (hunk 1 above)
Comment (pnicolucci) [reply dev/com.ibm.ws.session/resources/OSGI-INF/l10n/metatype.properties]: @volosied  you have a spelling issue here `partitioned` not `partiton`

File: dev/com.ibm.ws.transport.http/src/com/ibm/ws/http/channel/internal/HttpBaseMessageImpl.java [hunk 2]
```diff
@@ -2939,7 +2939,7 @@ else if (getServiceContext().getHttpConfig().onlySameSiteStar()) {
                 }
             }
             
-            // If SameSite=None is set programmatically, but partitioned is set via server.xml, then add the parititioned attribute
+            // If SameSite=None is set programmatically, but partitioned is set via server.xml, then add the partititioned attribute
```
Comment (pnicolucci) [inline dev/com.ibm.ws.transport.http/src/com/ibm/ws/http/channel/internal/HttpBaseMessageImpl.java]: `partitioned`

**Comment by volosied** [comment]
#build #spawn.fullfat.buckets=io.openliberty.webcontainer.servlet.6.0.internal_fat,com.ibm.ws.webcontainer.servlet.4.0_fat

**Comment by volosied** [comment]
#build #spawn.fullfat.buckets=io.openliberty.webcontainer.servlet.6.0.internal_fat,com.ibm.ws.webcontainer.servlet.4.0_fat

**Comment by volosied** [comment]
#build

**Comment by volosied** [comment]
#build #spawn.fullfat.buckets=io.openliberty.webcontainer.servlet.6.0.internal_fat,com.ibm.ws.webcontainer.servlet.4.0_fat

**Comment by volosied** [comment]
#build #spawn.fullfat.buckets=io.openliberty.webcontainer.servlet.6.0.internal_fat,com.ibm.ws.webcontainer.servlet.4.0_fat,com.ibm.ws.webcontainer.servlet.4.0_fat.2

**Comment by volosied** [comment]
#build #spawn.fullfat.buckets=io.openliberty.webcontainer.servlet.6.0.internal_fat,com.ibm.ws.webcontainer.servlet.4.0_fat,com.ibm.ws.webcontainer.servlet.4.0_fat.2

**Comment by pnicolucci** [comment]
@volosied  please rebase and let us keep a current PB so we can merge when we get approvals. Thanks!

**Comment by volosied** [comment]
#build

**Comment by volosied** [comment]
#build #spawn.fullfat.buckets=io.openliberty.webcontainer.servlet.6.0.internal_fat,com.ibm.ws.webcontainer.servlet.4.0_fat,com.ibm.ws.webcontainer.servlet.4.0_fat.2

**Comment by volosied** [comment]
#build #spawn.fullfat.buckets=io.openliberty.webcontainer.servlet.6.0.internal_fat,com.ibm.ws.webcontainer.servlet.4.0_fat,com.ibm.ws.webcontainer.servlet.4.0_fat.2

**Comment by volosied** [comment]
#build #spawn.fullfat.buckets=io.openliberty.webcontainer.servlet.6.0.internal_fat,com.ibm.ws.webcontainer.servlet.4.0_fat,com.ibm.ws.webcontainer.servlet.4.0_fat.2

File: dev/com.ibm.ws.webcontainer.security/src/com/ibm/ws/webcontainer/security/internal/WebAppSecurityConfigImpl.java [hunk 3]
```diff
@@ -202,8 +202,10 @@ public WebAppSecurityConfigImpl(Map<String, Object> newProperties,
         String partValue = (String) newProperties.get(CFG_KEY_PARTITIONED_COOKIE);
         if ("true".equalsIgnoreCase(partValue)||"false".equalsIgnoreCase(partValue)) {
             //we want partitionedCookie to be null unless the value is true or false
+            // defer is default which mean channel config determines the partitioned value
+            // if the value is true / false then this config is explicltly
```
Comment (pnicolucci) [inline dev/com.ibm.ws.webcontainer.security/src/com/ibm/ws/webcontainer/security/internal/WebAppSecurityConfigImpl.java]: explicitly what?

**Review by pnicolucci — APPROVED** [review]

**Comment by volosied** [comment]
#build #spawn.fullfat.buckets=io.openliberty.webcontainer.servlet.6.0.internal_fat,com.ibm.ws.webcontainer.servlet.4.0_fat,com.ibm.ws.webcontainer.servlet.4.0_fat.2

**Comment by volosied** [comment]
Only 2 failures -- which are existing defects.

Buildcon 4.

Merging!

**Pull request merged**